src/ast/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+add_library(verona-ast-lib
+  ast.cc
+  err.cc
+  files.cc
+  parser.cc
+  sym.cc
+  )
+
+target_link_libraries(verona-ast-lib CLI11::CLI11)
+target_link_libraries(verona-ast-lib cpp-peglib)
```

Comment:
peglib crashes with a `std::system_error` from inside `std::call_once` on Linux systems if it's linked without pthread.